doc/man3/PKCS5_PBE_keyivgen.pod

```diff
@@ -165,6 +171,10 @@ PKCS5_pbkdf2_set_ex() were added in OpenSSL 3.0.
 From OpenSSL 3.0 the PBKDF1 algorithm used in PKCS5_PBE_keyivgen() and
 PKCS5_PBE_keyivgen_ex() has been moved to the legacy provider as an EVP_KDF.
 
+In OpenSSL 3.2 the default salt length changed from 8 bytes to 16 bytes for PBE2
+related functions such as PKCS5_pbe2_set() if the I<saltlen> is 0.
```

Comment:
Drop the "if the saltlen is 0"?  Or make it a separate sentence: "The default is used if the passed saltlen is 0"?

The sentence is kind of confusing as it stands.